dev/conductor/core/lib/src/start.dart

```diff
@@ -60,6 +60,11 @@ class StartCommand extends Command<void> {
       help: 'The target release channel for the release.',
       allowed: kBaseReleaseChannels,
     );
+    argParser.addOption(
+      kFrameworkMirrorOption,
```

Comment:
We would also need an engine mirror.

Reply:
Mind if we consider this pr to just be framework since that is what was required for me to ship this week?

Reply:
I am okay with it as long as we have an open issue or PR to follow up with the engine changes.

Reply:
https://github.com/flutter/flutter/issues/154414